docs/development/modernizing-existing-add-ons.md

```diff
@@ -0,0 +1,68 @@
+<!--
+    This source file is part of the open source project
+    ExpressionEngine User Guide (https://github.com/ExpressionEngine/ExpressionEngine-User-Guide)
+
+    @link      https://expressionengine.com/
+    @copyright Copyright (c) 2003-2022, Packet Tide, LLC (https://packettide.com)
+    @license   https://expressionengine.com/license Licensed under Apache License, Version 2.0
+-->
+
+# Modernizing Existing Add-ons
+
+
+
+The ExpressionEngine 7.2 release brough a new add-on developemnt approach, which generates the majority of the add-ons needed files and struture.  It also bring additional organization to how add-ons are developed.
+
+NOTE: These changes do NOT break existing add-ons.  The old development methodologies will still work.
+
+The idea of a plugin, module, extension, fieldtype, etc. no longer exist.  Instead an add-on can have a template tag, or a field-type, or an extention.
+
+In order to harness the new add-on development methodology, you will have to do a few SMALL updates to your existing add-on's.
+
+TIP: You don't have to migrate your add-on to the new development methodology.  After making the following changes, you can leave your add-ons as is, and start new / additional development in the new method.
```

Comment:
Move this into the paragraph above.

Reply:
@dougblackjr I merged these two sentences and reworded them a bit. What do you think of the rewording?